lib/kubernetes-deploy/runner.rb

```diff
@@ -112,6 +112,24 @@ def template_variables
 
     private
 
+    # Inspect the file referenced in the kubectl stderr
+    # to make it easier for developer to understand what's going on
+    def inspect_kubectl_out_for_files(stderr)
+      # Output example:
+      # error: unable to decode "/tmp/path/to/file": [pos 96]: json: expect char '"' but got char '{'
+      match = stderr.match(/error: unable to decode "(?<path>\S+)":+/)
```

Comment:
Why `:+` at the end of the regex? Even if there were more than one colon somehow, why should the regex cover it?